js/background.js

```diff
@@ -1194,9 +1177,9 @@
       }
     });
 
-    Whisper.events.on('inviteFriends', async groupConvo => {
+    Whisper.events.on('inviteContacts', async groupConvo => {
       if (appView) {
-        appView.showInviteFriendsDialog(groupConvo);
+        appView.showInviteContactsDialog(groupConvo);
```

Comment:
I did not rename the file nor the class for now of this dialog (still called `InviteFriendDialog.tsx`). Should I include this rename in this PR?